src/index.html

```diff
@@ -11,6 +11,51 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action=""
+    method="GET"
+    data-qa="big"
+    >
+      <div class="container">
+        <label for="big-search">
+          <input
+            type="text"
+            name="big-search"
+            id="big-search"
+            class="input-field"
+            data-qa="keypress"
+            placeholder="Try &quot;Los-Angeles&quot;"
+          >
+          <img
+            src="images/Search.svg"
+            alt="search"
+            class="icon"
```

Comment:
```suggestion
            class="icon big-icon"
```
You could use the mix of classes here. Give `position: absolute` to `icon` class and two other classes will be responsible only for sizing and positioning